modules/rds/main.tf

```diff
@@ -0,0 +1,39 @@
+module "rds" {
+  #checkov:skip=CKV_TF_1: Use specific version instead of defining the commit hash
+  source  = "terraform-aws-modules/rds-aurora/aws"
+  version = "7.6.0"
```

Comment:
let's use the latest version ([8.5](https://registry.terraform.io/modules/terraform-aws-modules/rds-aurora/aws/latest))

Reply:
It was updated in 6f55c01 ✅